programs/gzip.c

```diff
@@ -32,14 +32,11 @@
 #include "prog_util.h"
 
 #include <errno.h>
-#include <sys/types.h>
 #include <sys/stat.h>
 #ifdef _WIN32
 #  include <sys/utime.h>
 #else
-#  include <sys/time.h>
```

Comment:
`sys/time.h` and `utime.h` are sometimes needed, see `restore_timestamps()`.

Reply:
My bad, I fixed it. I think the other headers can be safely removed.